apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/controllers/interface_controller.ex

```diff
@@ -24,6 +24,16 @@ defmodule Astarte.RealmManagement.APIWeb.InterfaceController do
 
   action_fallback Astarte.RealmManagement.APIWeb.FallbackController
 
+  def index(conn, %{"realm_name" => realm_name, "detailed" => "true"}) do
+    with {:ok, interfaces} <-
+           Astarte.RealmManagement.API.Interfaces.list_interfaces(realm_name, %{
+             "detailed" => true
+           }) do
+      decoded_interfaces = Enum.map(interfaces, &Jason.decode!/1)
```

Comment:
The `interfaces` are a JSON array already, no need to decode them (and let `render` encode them once again under the hood)

Reply:
Removed line for decoding list.